rpmlint/checks/KMPPolicyCheck.py

```diff
@@ -0,0 +1,43 @@
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class KMPPolicyCheck(AbstractCheck):
+    def check(self, pkg):
+        if pkg.isSource() or pkg.name.find('-kmp-') < 0:
+            return
+
+        pkg_requires = set(map(lambda x: x[0].split('(')[0], pkg.requires()))
+
+        kernel_flavour = "kernel-" + pkg.name.partition('-kmp-')[2]
```

Comment:
Please consistently use `flavor` here (American English is used throughout rpmlint)

Reply:
sure.